doc/source/guide/geometry_guide.rst

```diff
@@ -0,0 +1,240 @@
+.. _geometry_guide:
```

Comment:
I'm hesitant to make (another) subdirectory with only one file inside. I suggest we wait with this until there are more tomography-related guides.